lib/helpers.ts

```diff
@@ -1,7 +1,15 @@
 export const urlEncodeBase64 = (signature: string): string => {
-  signature = signature.replace(/(=+)$/g, '');
-  signature = signature.replace(/\//g, '_');
-  signature = signature.replace(/\+/g, '-');
+  signature = signature.replace(/(=+)$/g, "");
+  signature = signature.replace(/\//g, "_");
+  signature = signature.replace(/\+/g, "-");
 
   return signature;
 };
+
+export const urlSafeBase64ToBase64 = (signature) => {
+  if (isUrlSafeBase64(signature))
+    return signature.replace(/-/g, "+").replace(/_/g, "/");
+};
```

Comment:
Could you please apply this style?
```suggestion
  if (isUrlSafeBase64(signature)) {
    return signature.replace(/-/g, '+').replace(/_/g, '/');
  }
};
```

Also, what happens if the if statement returns false? Do you not need to return the input signature in that case?

```
export const urlSafeBase64ToBase64 = (signature) => {
  if (isUrlSafeBase64(signature)) {
    return signature.replace(/-/g, '+').replace(/_/g, '/');
  }

  return signature;
}

Reply:
Well😄, I'll resubmit it as soon as I can.